workflow/defaults/default_resources.yaml

```diff
@@ -11,7 +11,7 @@
 
 godas_resource_table:
 #                ranks  ppn  wallclock              threads MB
-  r3dvar:       [  100,   20, !timedelta "01:00:00", 12,  "3072" ]
+  r3dvar:       [  80,   20, !timedelta "01:00:00", 12,  "3072" ]
```

Comment:
Add one exclusive entry for the envar, and for now, we can set the r3dvar:       [  20,   20, ...].